.github/workflows/main.yml

```diff
@@ -0,0 +1,81 @@
+# This GitHub Actions workflow contains two jobs:
+# -- First, Lint uses the latest Ubuntu image and Python 3.9 to check
+# the code and writing for defects, using the Poetry lint task
+#
+# -- Second, Test uses a strategy matrix to test the code with different
+# Python versions and determine code coverage, using the Poetry test task
+
+name: Lint and Test
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Python 3.9
+        uses: actions/setup-python@v2
+        id: setup-python
+        with:
+          python-version: 3.9
+      - name: Install Poetry
+        uses: abatilo/actions-poetry@v2.1.4
+        with:
+          poetry-version: 1.1.13
+      - name: Setup Poetry
+        run: |
+          poetry config virtualenvs.create true
+          poetry config virtualenvs.in-project true
+          poetry env info
+      - name: Install dependencies
+        run: poetry install --no-interaction --no-ansi
+      - name: Lint code
+        run: poetry run task lint
```

Comment:
Usually, when setting up the taskipy tasks to run linters, we use the `lint` task to run all linters. But this causes a small problem when setting it up on Actions. Here is an example:  `poetry run task lint` typically exists on the first linter that has an issue and does not run the rest so you would have to fix the error, rerun the task to see the remaining lint errors. To show all errors at the same time, I think this should be split up to multiple steps, one for each linter. This way, you would guarantee that all linters run on every Actions job. Lastly, you can add a ` if: always()` on each linting step to make sure it runs even if previous ones fail.

Reply:
Although `if: always()` will work, there's a more semantically accurate way to specify this for individual steps in a job: [`continue-on-error: true`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error).

Reply:
This has since been changed so that they are all separate jobs as per @mariakimheinert feedback